modules/custom/d_p/js/d_p_ckeditor.js

```diff
@@ -7,33 +7,46 @@
         d_p_ckeditor();
       });
 
-      function d_p_ckeditor_add_js(doc) {
-        var cssId = 'd_p_ckeditor';
-        if (!doc.getElementById(cssId)) {
-          // add css for cke_editable class in ckeditor
-          var head = doc.getElementsByTagName('head')[0];
-          var link = doc.createElement('link');
-          link.id = cssId;
-          link.rel = 'stylesheet';
-          link.type = 'text/css';
-          link.href = '/profiles/contrib/droopler/modules/custom/d_p/css/d_p_ckeditor.css';
-          head.appendChild(link);
-        }
-      }
-
+      // @todo: Rework this to better solution when it will be possible.
       function d_p_ckeditor() {
-        if (typeof CKEDITOR !== "undefined") {
-          for (let instanceName in CKEDITOR.instances) {
-            if (CKEDITOR.instances[instanceName].element.$.parentElement.classList.contains('d_p_ckeditor_centered')) {
-              if (typeof CKEDITOR.instances[instanceName].document !== 'undefined') {
-                d_p_ckeditor_add_js(CKEDITOR.instances[instanceName].document.$);
-              } else {
-                setTimeout(function () {
-                  d_p_ckeditor_add_js(CKEDITOR.instances[instanceName].document.$);
-                }, 500);
+        if (typeof Drupal.CKEditor5Instances !== "undefined") {
+          $(once('d_p_ckeditor_centered', '.d_p_ckeditor_centered', context)).each(function () {
+            let editor_id = $(this).find('textarea').attr('data-ckeditor5-id');
+            let editor = Drupal.CKEditor5Instances.get(editor_id);
+            let $editor_editable = $(this).find('.ck-editor__editable');
+
+            // Set default alignment to center.
+            $editor_editable.children().each(function () {
+              if (!$(this).hasClass('text-align-left') && !$(this).hasClass('text-align-right') && !$(this).hasClass('.text-align-justify')) {
+                $(this).addClass('text-align-center');
               }
-            }
-          }
+            });
+            editor.setData($editor_editable.html());
+
+            // Add text-align-left class to selected element.
+            $(once('open_alignment_dropdown', '.ck.ck-dropdown.ck-toolbar-dropdown.ck-alignment-dropdown')).on('click', function() {
+              $(once('alignment_select', '.ck.ck-dropdown.ck-toolbar-dropdown.ck-alignment-dropdown .ck.ck-toolbar__items button')).on('click', function() {
+                let current_alignment = editor.commands.get('alignment').value;
+                if (current_alignment === 'left') {
+                  editor.editing.view.change(writer => {
+                    if (typeof editor?.editing?.view?.document?.selection?.focus?.parent?.parent !== 'undefined') {
+                      writer.addClass('text-align-left', editor.editing.view.document.selection.focus.parent.parent);
```

Comment:
I had to use this "editor.editing.view.document.selection.focus.parent.parent"
because "editor.editing.view.document.selection.getSelectedElement()" returns null, any idea why?
Also maybe there is some way to loop through all selected elements?

Reply:
`getSelectedElement()` is not "the element in which there's selection", but "the element that is being selected from outside" – e.g. when there's an image selected.  
  
There are no useful methods on the view Selection API because it should be used very rarely. 99% of plugin code should operate on the model selection. And on the model, getting all the blocks to which the alignment change should be applied is far more straightforward: https://github.com/ckeditor/ckeditor5/blob/4854443c0ab304833cfeba302faf458f1e2a5243/packages/ckeditor5-alignment/src/alignmentcommand.ts#L68